macOS/DuckDuckGo/Preferences/View/PreferencesDataClearingView.swift

```diff
@@ -47,6 +48,20 @@ extension Preferences {
                         PreferencePaneSubSection {
                             ToggleMenuItem(UserText.openFireWindowByDefault, isOn: $model.openFireWindowByDefault)
                                 .accessibilityIdentifier("PreferencesDataClearingView.openFireWindowByDefault")
+
+                            if model.openFireWindowByDefault && startupModel.restorePreviousSession {
```

Comment:
Sorry for being a pain. I wasn’t reviewing the "open Fire Window by default” PR, but this should be called `shouldOpenFireWindowbyDefault` to make it clear that it’s a boolean value and not a function.

And I know it’s not your code, but `restorePreviousSession` should be called `shouldRestorePreviousSession` but I suppose we have to live with it :)